exotica_python/src/pyexotica/__init__.py

```diff
@@ -4,4 +4,6 @@
 from .publish_trajectory import *
 from .tools import *
 from .interactive_cost_tuning import *
+
+Visualization = VisualizationMoveIt
```

Comment:
Add note this is for backwards-compatibility, we will remove in future. If there is a deprecated decorator in Python, I think there is, use that one

Reply:
Can't deprecate it properly without extra wrappers and stuff. Just adding a comment.